example/src/app/components/malware-modal/malware-modal.component.html

```diff
@@ -0,0 +1,35 @@
+<ng-container *ngIf="!isDisabled">
```

Comment:
Ak to je na celú komponentu, tak by to podľa mňa malo byť v parent HTML template